modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/test/js/designer/definition-builder/diagram-builder/components/sidebar/sections/NodeInformationBaseSection.spec.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {render} from '@testing-library/react';
+
+import '@testing-library/jest-dom/extend-expect';
+import React from 'react';
+
+import {NodeInformationBaseSection} from '../../../../../../../../src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/NodeInformationBaseSection'; // Adjust path as needed
```

Comment:
hi @danvb112, this comment should be here?

Reply:
Ops, my bad! Thanks @rebsilva!